src/assets/images/iconDonate.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="48" height="48" viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M24 36.5C32.9766 36.5 40.25 29.2266 40.25 20.25C40.25 11.2734 32.9766 4 24 4C15.0234 4 7.75 11.2734 7.75 20.25C7.75 29.2266 15.0234 36.5 24 36.5ZM22.2657 11.6093V10.2968C22.2657 9.5781 22.8438 8.99997 23.5626 8.99997H24.4297C25.1485 8.99997 25.7266 9.5781 25.7266 10.2968V11.625C26.9376 11.6875 28.1094 12.1015 29.086 12.8281C29.5235 13.1484 29.5704 13.789 29.1797 14.164L27.9063 15.375C27.6094 15.664 27.1641 15.6718 26.8126 15.4531C26.3907 15.1875 25.9219 15.0547 25.4219 15.0547H22.3829C21.6797 15.0547 21.1094 15.6953 21.1094 16.4843C21.1094 17.125 21.5001 17.6953 22.0547 17.8593L26.9219 19.3203C28.9297 19.9218 30.336 21.8515 30.336 24.0156C30.336 26.6718 28.2735 28.8203 25.7188 28.8906V30.2031C25.7188 30.9218 25.1407 31.5 24.4219 31.5H23.5547C22.836 31.5 22.2579 30.9218 22.2579 30.2031V28.875C21.0469 28.8125 19.8751 28.3984 18.8985 27.6718C18.461 27.3515 18.4141 26.7109 18.8047 26.3359L20.0782 25.125C20.3751 24.8359 20.8204 24.8281 21.1719 25.0468C21.5938 25.3125 22.0626 25.4453 22.5626 25.4453H25.6016C26.3047 25.4453 26.8751 24.8047 26.8751 24.0156C26.8751 23.375 26.4844 22.8047 25.9297 22.6406L21.0626 21.1797C19.0547 20.5781 17.6485 18.6484 17.6485 16.4843C17.6563 13.8281 19.711 11.6797 22.2657 11.6093ZM41.5 31.5H38.9609C37.4297 33.5312 35.4766 35.2265 33.2578 36.5H38.2422C38.6563 36.5 38.9922 36.7812 38.9922 37.125V38.375C38.9922 38.7187 38.6563 39 38.2422 39H9.75C9.33594 39 9 38.7187 9 38.375V37.125C9 36.7812 9.33594 36.5 9.75 36.5H14.7344C12.5156 35.2265 10.5703 33.5312 9.03125 31.5H6.5C5.11719 31.5 4 32.6172 4 34V41.5C4 42.8828 5.11719 44 6.5 44H41.5C42.8828 44 44 42.8828 44 41.5V34C44 32.6172 42.8828 31.5 41.5 31.5Z" fill="#858199"/>
```

Comment:
I think we need to set the fill to `#000` to allow us to change it programmatically for dark mode

Reply:
ahhh, thanks for the context